build/Jenkinsfile.branch

```diff
@@ -125,17 +95,15 @@ pipeline {
           platformStages.platformCheckoutTE(env.BRANCH_NAME)
         }
         dir('src/github.com/mattermost/mattermost-server') {
-          withCredentials([usernamePassword(credentialsId: 'matterbuild-docker-hub', usernameVariable: 'DOCKER_USER', passwordVariable: 'DOCKER_PASS')]) {
+          withCredentials([usernamePassword(credentialsId: 'matterbuild-docker-hub', usernameVariable: 'DOCKER_USER', passwordVariable: 'DOCKER_PASS'), string(credentialsId: 'mattermost-enterprise-docker-trust-pass', variable: 'DCT_ENTERPRISE_PASS')]) {
             sh 'docker login --username ${DOCKER_USER} --password ${DOCKER_PASS}'
             sh """
-            export TAG='${env.BRANCH_NAME}'
-            docker build --no-cache --build-arg MM_PACKAGE=https://releases.mattermost.com/mattermost-platform/${env.BRANCH_NAME}/mattermost-enterprise-linux-amd64.tar.gz -t mattermost/mattermost-enterprise-edition:\${TAG} build
-            docker push mattermost/mattermost-enterprise-edition:\${TAG}
-            """
-            sh """
-            export TAG='${env.BRANCH_NAME}'
-            docker build --no-cache --build-arg MM_PACKAGE=https://releases.mattermost.com/mattermost-platform/${env.BRANCH_NAME}/mattermost-team-linux-amd64.tar.gz -t mattermost/mattermost-team-edition:\${TAG} build
-            docker push mattermost/mattermost-team-edition:\${TAG}
+            export TAG=$(git rev-parse --short=7 HEAD)
+            docker build --no-cache --build-arg MM_PACKAGE=https://releases.mattermost.com/mattermost-platform/${env.BRANCH_NAME}/mattermost-enterprise-linux-amd64.tar.gz -t mattermost/cloud-beta-ee:\${TAG} -t mattermost/cloud-beta-ee:latest build
+
+            export DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE=${DCT_ENTERPRISE_PASS}
+            docker push mattermost/cloud-beta-ee:\${TAG}
+            docker push mattermost/cloud-beta-ee:latest
```

Comment:
maybe just `mattermost/mm-cloud`  because we might use that to non-beta as well 0/5